src/main/resources/assets/admin/common/js/content/Widget.ts

```diff
@@ -43,6 +44,18 @@ export class Widget<B extends WidgetBuilder = WidgetBuilder, C extends WidgetCon
         return this.iconUrl;
     }
 
+    public getFullUrl(): string {
+        return `${this.getBaseUrl()}/${this.url}`;
+    }
+
+    public getFullIconUrl(): string {
+        return `${this.getBaseUrl()}${this.iconUrl}`;
+    }
+
+    private getBaseUrl(): string {
+        return (CONFIG.getString('widgetApiUrl') || '').replace(/\/+$/, '');
```

Comment:
Since it is not possible to mount `/_/` on `/` there should not be necessary to remove trailing slash ever